Day-I-Social-Card/social-card/src/components/HeaderComponents/HeaderContainer.js

```diff
@@ -1,2 +1,15 @@
 import React from 'react';
-import './Header.css';
+import ImageThumbnail from './ImageThumbnail';
+import HeaderTitle from './HeaderTitle';
+import HeaderContent from './HeaderContent';
+const HeaderContainer =()=>{
+    return(
+        <div>
+            <h1>{ImageThumbnail}</h1>
```

Comment:
When you want to place a component you can just put use it like `< ImageThumbnail />`

Reply:
oh seriously